cpp/arcticdb/entity/types.hpp

```diff
@@ -115,6 +115,8 @@ enum class ValueType : uint8_t {
 
     UTF_DYNAMIC = 11,
     ASCII_DYNAMIC = 12,
+    EMPTY = 13,
```

Comment:
It is in fact equivalent to None type in Python. I will add a comment in code. @willdealtry used this notation (Empty type/EmptyVal) so I went along with it as well. Happy to change it if people think it would be more readable otherwise.

Reply:
Could be NoneType I suppose. The purpose is to indicate that there's a column there but it doesn't have a defined type in a particular segment. In the case of Python, it's probably that all of the elements are None, but it's intended to be more generally applicable.